tests/inc/options/test-class-wpseo-options.php

```diff
@@ -99,4 +99,38 @@ public function test_get_option_IS_VALID_with_valid_option_name() {
 		$result = WPSEO_Options::get_option( 'wpseo' );
 		$this->assertArrayHasKey( 'website_name', $result );
 	}
+
+	/**
+	 * Tests if the get_option_value function returns a valid result.
+	 *
+	 * @covers WPSEO_Options::get_option_value()
+	 */
+	public function test_get_option_value_returns_valid_result() {
+		$result = WPSEO_Options::get_option_value( 'wpseo', 'keyword_analysis_active' );
```

Comment:
This test is tricky, because of we aren't sure what value `keyword_analysis_active` will have. I think it is better to set the value first. If you are testing against the option defaults you might consider running the test in a separate process: https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.runInSeparateProcess

Reply:
Prefer to just explicitly set the value instead of assuming the default is untouched (and unchanged in the future)

Reply:
I'll force them back to their defaults first.